pkg/scheduler/framework/framework.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package framework features the scheduler framework, which the scheduler runs to schedule
+// a placement to most appropriate clusters.
+package framework
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/client-go/tools/record"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	fleetv1 "go.goms.io/fleet/apis/v1"
+	"go.goms.io/fleet/pkg/scheduler/framework/parallelizer"
+)
+
+const (
+	// eventRecorderNameTemplate is the template used to format event recorder name for a scheduler framework.
+	eventRecorderNameTemplate = "scheduler-framework-%s"
```

Comment:
is this event msg readable for user?

Reply:
Hi Ryan! Ah, this isn't the event message, but the name of the event recorder in use by the scheduler framework.